doc/gh-pages/pages/development/qtwasmtutorial.md

```diff
@@ -61,26 +61,25 @@ nav_order: 3
       ./init-repository -f --module-subset=qtbase,qtcharts,qtsvg
       ```
 
-    * Navigate to parent directory, create new shadow build folder and cd into it:
+    * Navigate to parent directory, create new shadow build folder:
 
       ```
       cd ..
       mkdir qt5_shadow
-      cd qt5_shadow
       ```
 
-    * Call configure from new working directory in order to perform a shadow build.
+    * Call configure from new working directory in order to perform a shadow build. This option is preferable in order to keep Qt source tree clean from this build, which will be stored in a separate directory (prefix), especially if you want to have multiple builds, in order to test for different versions or configurations.
 
       With thread support:
 
       ```
-      ../qt5/configure -opensource -confirm-license -xplatform wasm-emscripten -feature-thread -nomake examples -no-dbus -no-ssl -prefix /home/lorenz/Qt/5.14.0/wasm_em1393_64_withThread
+      ../qt5/configure -opensource -confirm-license -xplatform wasm-emscripten -feature-thread -nomake examples -no-dbus -no-ssl -prefix /$PWD/qt5_shadow/wasm_em1393_64_withThread
```

Comment:
This will install the compiled qt version into the qt5_shadow folder and mix it with the temporary build objects etc. This is why we cd into the qt_shadow build folder call qmake from there and install to a different location. How about if we aggree on the following folder structure for this tutorial?

parent/
├── mne-cpp/
├── qt5/
├── qt5_shadow/
└── qt5_wasm_em1393_64_withThread/

Reply:
I say that looks fine. I'm having trouble compiling a demo project I made for wasm. Please, let me think about it for a day or so.